### PR TITLE
Remove redundancy code in golang-pkg (#108)

### PR DESCRIPTION
**Pull request by xiaoch05**

### CODE REVIEW

**Comment by clearloop** [comment]
Have this pr done? If so, I can merge this, because a modification I'm working on will remove all the config of golang

File: pkg/shadow/config.go [hunk 1]
```diff
@@ -3,27 +3,18 @@ package shadow
...
-
 	"github.com/darwinia-network/shadow/pkg/shadow/util"
 )
 
 const (
-	ETHEREUM_ROPSTEN     string = "ETHEREUM_ROPSTEN"
-	ETHEREUM_RPC         string = "ETHEREUM_RPC"
-	SHADOW_GENESIS       string = "SHADOW_GENESIS"
-	GETH_DATADIR         string = "GETH_DATADIR"
 	DEFAULT_ETHEREUM_RPC string = "https://mainnet.infura.io/v3/0bfb9acbb13c426097aabb1d81a9d016"
```
Comment (HackFisher) [inline pkg/shadow/config.go:10]: ```DEFAULT_ETHEREUM_RPC ``` need to be removed too, and removed the code using ```DEFAULT_ETHEREUM_RPC ``` in header.go as well.

**Review by HackFisher — CHANGES_REQUESTED** [review]

File: pkg/shadow/config.go (hunk 1 above)
Comment (xiaoch05) [reply pkg/shadow/config.go]: removed (https://github.com/darwinia-network/shadow/commit/8c21c2506683b2d98c82c0ddba85c0d08786b354)

**Review by clearloop — APPROVED** [review]
Please add `A0-pleasereview` label when you completing PRs.

**Review by HackFisher — APPROVED** [review]

**Pull request merged**